web/blueprints/finance/__init__.py

```diff
@@ -548,6 +552,66 @@ def bank_account_activities_match() -> ResponseReturnValue:
                            activities_team=matched_activities_team)
 
 
+class ActivityEntry(t.TypedDict):
+    bank_account: str
+    name: str
+    valid_on: date
+    reference: str
+    amount: int
+
+
+@bp.route("/bank-account-activities/return/")
+@access.require("finance_change")
+def bank_account_activities_return() -> ResponseReturnValue:
+    field_list: BooleanFieldList = []
+    activities: dict[str, ActivityEntry] = {}
+
+    for activity in get_activities_to_return():
+        activities[str(activity.id)] = {
+            "bank_account": activity.bank_account.name,
+            "name": activity.other_name,
+            "valid_on": activity.valid_on,
+            "reference": activity.reference,
+            "amount": activity.amount,
+        }
+
+        field_list.append((str(activity.id), BooleanField(str(activity.id), default=True)))
+
+    form: t.Any = _create_form(field_list)
+
+    return render_template(
+        "finance/bank_account_activities_return.html",
+        form=form(),
+        activities=activities,
+    )
+
+
+@bp.route("/bank-account-activities/return/do/", methods=["GET", "POST"])
```

Comment:
The `"GET"` is just there so the `tests/frontend/test_permissions.py:test_access_finance` receives a `403` instead of a `405`.  Do you think that's fine?

Reply:
(or should the test be adjusted?)

Reply:
The test, whose implicit assumption is that every endpoint is `GET`, should definitely be adjusted, as this assumption is wrong.